multi-node/aws/pkg/config/config.go

```diff
@@ -329,6 +331,18 @@ func (cfg Cluster) valid() error {
 	if cfg.ExternalDNSName == "" {
 		return errors.New("externalDNSName must be set")
 	}
+
+	releaseChannelSupported := false
+	for _, channel := range supportedReleaseChannels {
```

Comment:
Could use a map for supportedReleaseChannels and would make this check a little cleaner.